index.html

```diff
@@ -7,6 +7,7 @@
       src="https://www.w3.org/Tools/respec/respec-w3c"
       class="remove"
     ></script>
+    <script class="remove" src="https://cdn.jsdelivr.net/gh/transmute-industries/respec-vc-jwt@0.0.7/dist/main.js"></script>
```

Comment:
Should this be hosted by w3c?

Reply:
Yes, thanks for spotting this, @andresuribe87. The external script references to the specification text should be in the same directory (or subdirectory) relative to the spec text itself, and refer to it via relative URL.

Reply:
I'm happy to fix this after it is fixed in the vc data model:

https://github.com/w3c/vc-data-model/blob/main/index.html#L14